### PR TITLE
[site,util] Site build fixes

### PR DESCRIPTION
**Pull request by hcallahan-lowrisc**
Invoke bazel without creating the convenience symlinks when building the documentation site. This is one way of working around the mdbook issue where excess files are copied into the build directory, bloating the deployed size.

Also, since we now use bazel to build some rust projects within the site build and there is no cache used between jobs, bump up the number of vCPUs available to stop the build from timing out.

### CODE REVIEW

File: util/site/build-docs.sh
Comment (HU90m) [inline util/site/build-docs.sh:175]: Nit: I would find this easier to read

```suggestion
    bazel_out="$(./bazelisk.sh info output_path 2>/dev/null)"
     #TODO : get the target's path using cquery
    target_rustdoc_output_path="$bazel_out/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc"
```

**Comment by hcallahan-lowrisc** [comment]
> This looks pretty good and builds fine. Why is it a draft?

Just so I could test it by running the build manually from Gcloud, pointing it at this PR.

File: util/site/build-docs.sh
Comment (hcallahan-lowrisc) [reply util/site/build-docs.sh]: Done

**Review by marnovandermaas — APPROVED** [review]
Thanks, looks good to me. I'll wait for CI to pass before merging.

**Review by HU90m — APPROVED** [review]

**Comment by marnovandermaas** [comment]
CI failures are due to bitstream building problems that are affecting multiple PRs. I will merge this PR because it fixes the `site-builder` CI task.

**Pull request merged**
